step.yml

```diff
@@ -1,4 +1,4 @@
-title: "[BETA] Xcode Build for testing for iOS"
```

Comment:
Don't forget to reformat this file according to our latest YML indentation guideline before merging the PR!

Reply:
updated in https://github.com/bitrise-steplib/steps-xcode-build-for-test/pull/17/commits/d259f005d2b1e6d62b7a35db62602ca778c3e5a0